src/Sylius/Bundle/ApiBundle/Command/Admin/ResetPassword.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Command\Admin;
```

Comment:
```suggestion
namespace Sylius\Bundle\ApiBundle\Command\AdminUser;
```
But I'm not convinced 🤔